_posts/2016/2016-07-07-wiremock.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "How not to produce 'Distributed Balls of Mud'"
+date: "2016-07-07 12:00:00"
+image: /blog-header/balls-of-mud.jpg
+categories: tech-stories
+authors: ["Oliver"]
+---
+
+There is a lot of buzz about microservices these days, and here is our take on managing decoupled microservices, while still keeping confident in REST API compatibility.
+
+One of the SOA missteps was trying to enforce a canonical domain model, which lead to bloated domain objects, that still would not fit all use cases and needed some sort of central management.
+Just try to come up with a common model of any real-world object and you'll know that this does not scale the more people are involved.
+In a more abstract sense, it is our strive to perfection and the strive of perfect modeling of our problem domain that drove us into the wrong direction.
+David Dawson phrased this as "Perfection and How It Ruined The World" in his talk ["Philosophical Architecture in Grails"](https://www.youtube.com/watch?v=nx8XMY7evbA)
```

Comment:
You can also directly implement the video like that:

{% youtube nx8XMY7evbA %}